stl/inc/valarray

```diff
@@ -45,11 +45,9 @@ _Ty* _Allocate_for_op_delete(size_t _Count) {
     const size_t _Bytes = _Get_size_of_n<sizeof(_Ty)>(_Count);
 #ifdef __cpp_aligned_new
     constexpr bool _Extended_alignment = alignof(_Ty) > __STDCPP_DEFAULT_NEW_ALIGNMENT__;
-    if
-        _CONSTEXPR_IF(_Extended_alignment) {
-            return static_cast<_Ty*>(::operator new (_Bytes, align_val_t{alignof(_Ty)}));
-        }
-    else
+    if _CONSTEXPR_IF (_Extended_alignment) {
+        return static_cast<_Ty*>(::operator new (_Bytes, align_val_t{alignof(_Ty)}));
+    } else

```

Comment:
I suspect that reversing the clang-format damage here won't work; clang-format will want to put it back.

Reply:
The original formatting was odd though

Reply:
Yes, clang-format doesn't understand what the `_CONSTEXPR_IF` macro is doing here, and does something silly. We've decided we hate the poor formatting less than we hate the idea of wrapping every occurrence in `// clang-format off`.

Reply:
If you peek at the failed CI results (https://github.com/microsoft/STL/pull/286/checks?check_run_id=297012212) you'll see that clang-format wants this reverted.

Reply:
We reported this as LLVM-39248 which has been fixed in clang-format 10, so our suffering will soon end.